cips/cip-draft_ica.md

```diff
@@ -0,0 +1,233 @@
+  ...
+  paramsKeeper.Subspace(icahosttypes.SubModuleName)
+  ...
+}
+```
+
+### Host module parameters
+| Key                    | Type     | Default Value |
+|------------------------|----------|---------------|
+| `HostEnabled`          | bool     | `true`        |
+| `AllowMessages`        | []string | `["*"]`       |
+
+#### HostEnabled
+
+The `HostEnabled` parameter controls a chain's ability to service ICS-27 host specific logic. 
+
+#### AllowMessages
+
+The `AllowMessages` parameter provides the ability for a chain to limit the types of messages or transactions that hosted interchain accounts are authorized to execute by defining an allowlist using the Protobuf message type URL format.
+
+For example, a Cosmos SDK-based chain that elects to provide hosted Interchain Accounts with the ability of staking and unstaking will define its parameters as follows:
+
+```
+"params": {
+    "host_enabled": true,
+    "allow_messages": ["/cosmos.staking.v1beta1.MsgDelegate", "/cosmos.staking.v1beta1.MsgUndelegate"]
+}
+```
+
+There is also a special wildcard `"*"` value which allows any type of message to be executed by the interchain account. This must be the only value in the `allow_messages` array.
+
+```
+"params": {
+    "host_enabled": true,
+    "allow_messages": ["*"]
+}
+```
+
+## Rationale
+
+- Permissionless: An interchain account may be created by any actor without the approval of a third party (e.g. chain governance). Note: Individual implementations may implement their own permissioning scheme, however the protocol must not require permissioning from a trusted party to be secure.
+- Fault isolation: A controller chain must not be able to control accounts registered by other controller chains. For example, in the case of a fork attack on a controller chain, only the interchain accounts registered by the forked chain will be vulnerable.
+- The ordering of transactions sent to an interchain account on a host chain must be maintained. Transactions must be executed by an interchain account in the order in which they are sent by the controller chain.
+- If a channel closes, the controller chain must be able to regain access to registered interchain accounts by simply opening a new channel.
+- Each interchain account is owned by a single account on the controller chain. Only the owner account on the controller chain is authorized to control the interchain account. The controller chain is responsible for enforcing this logic.
+- The controller chain must store the account address of any owned interchain accounts registered on host chains.
+- A host chain must have the ability to limit interchain account functionality on its chain as necessary (e.g. a host chain can decide that interchain accounts registered on the host chain cannot take part in staking).
+
+## Backwards Compatibility
+
+This proposal is backwards-incompatible because it is state-machine breaking. The feature must be released introduced in a new major version.
```

Comment:
[nit] released and introduces seems duplicative

```suggestion
This proposal is backwards-incompatible because it is state-machine breaking. The feature must be introduced in a new major version.
```

Reply:
[[done](https://github.com/celestiaorg/CIPs/pull/39/commits/2d9148935c80ccbb9d236a3f7c00e94de76ea254)]